src/main/java/net/liquidpineapple/pang/screens/WinScreen.java

```diff
@@ -0,0 +1,91 @@
+package net.liquidpineapple.pang.screens;
+
+import lombok.extern.slf4j.Slf4j;
+import net.liquidpineapple.pang.Application;
+import net.liquidpineapple.pang.InputHandler;
+import net.liquidpineapple.pang.objects.NumberToken;
+
+
+import javax.imageio.ImageIO;
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Class that represents the winscreen.
+ * @author Jaap-Jan
+ * @date 21-9-2016.
+ */
+@Slf4j
+public class WinScreen extends Screen {
+    private static ArrayList<NumberToken> Places;
+    /**
+     * Constructor for the winscreen.
+     */
+    public WinScreen() {
+        try {
+            backgroundImage = ImageIO.read(Level.class.getResource("/sprites/winscreen.png"));
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        //These are from right to left.
+        Places = new ArrayList<>(9);
+        NumberToken nine = new NumberToken(524,300);
+        NumberToken eight = new NumberToken(492,300);
+        NumberToken seven = new NumberToken(460,300);
+        NumberToken six = new NumberToken(428,300);
+        NumberToken five = new NumberToken(396,300);
+        NumberToken four = new NumberToken(364,300);
+        NumberToken three = new NumberToken(332,300);
+        NumberToken two = new NumberToken(300,300);
+        NumberToken one = new NumberToken(268,300);
+        Places.add(nine);
+        Places.add(eight);
+        Places.add(seven);
+        Places.add(six);
+        Places.add(five);
+        Places.add(four);
+        Places.add(three);
+        Places.add(two);
+        Places.add(one);
+        objectList.add(nine);
+        objectList.add(eight);
+        objectList.add(seven);
+        objectList.add(six);
+        objectList.add(five);
+        objectList.add(four);
+        objectList.add(three);
+        objectList.add(two);
+        objectList.add(one);
```

Comment:
And this can be easily done with:
```
for (NumberToken token : Places()) {
            output.objectList.add(token);
        }

```
Both of these would also remove the need to name all of the numbertokens.